adb_graphics/default_specs.yml

```diff
@@ -1301,6 +1301,24 @@ trc1:
     title: Vertically Integrated Smoke
     transform: conversions.to_micro
     unit: $mg/m^2$
+  lvl5:
```

Comment:
I think that if you set the level here to `1000ft` and `6000ft` below, then you shouldn't need to provide a vertical_index. I've run into issues providing vertical index explicitly, especially when different models/configs output different sets of levels.

I'm not sure what the value of levels are in the vertical dimension. Maybe it's kilofeet? If needed, a multiplier can be applied inside the method `numeric_level` in the `datahandler.gribdata` package.

Reply:
So I discovered that I didn't need vertical_index because using the names "lvl5" and "lvl12" were parsed making the levels 5 and 12.  But the actual level indices that I need are 4 and 11.  I changed these to "lvl4" and "lvl11" in the next commit.

The index of the vertical level in the native files doesn't have a unit.  The monikers "1000ft" and "6000ft" are approximations, so I can't provide that as a level.

It looks like the test_common.py gives me an assertion error related to "lvl4" in my latest test.  The graphics run fine, but the pytest generates this error.  Is this something that can be corrected in test_common.py?